cpp/meson.build

```diff
@@ -0,0 +1,19 @@
+project('Lift-Kata', 'cpp', default_options : ['cpp_std=c++17'])
+
+gtest_dep = dependency('gtest', main : false, required : false)
+gmock_dep = dependency('gmock', main : true, required : false)
+catch2_dep = dependency('catch2', fallback : ['catch2', 'catch2_dep'])
```

Comment:
How does Meson get ApprovalTests.cpp? If it doesn't, is this why there is a download of ApprovalTests.cpp in include?

Reply:
It doesn't, as ApprovalsTests doesn't have a meson.build system and I didn't think I would keep one up to date I aborted my tries to make a meson build wrapper for ApprovalsTests.